.mergify.yml

```diff
@@ -54,3 +54,11 @@ pull_request_rules:
       merge: {}
       dismiss_reviews: {}
       delete_head_branch: {}
+  - name: backport patches to release v0.1 branch
+    conditions:
+      - base=main
```

Comment:
shouldn't `base` be the release branch name here?

Reply:
If the PR is sent only to the main branch it will backport to release the v0.1 branch or else it will not.